src/toil/utils/toilDestroyCluster.py

```diff
@@ -26,6 +26,11 @@ def main():
     parser.add_argument('-p',"--provisioner", dest='provisioner', choices=['aws'], required=True,
                         help="The provisioner for cluster auto-scaling. Only aws is currently"
                              "supported")
+    parser.add_argument('-z', '--zone', dest='zone', required=False,
```

Comment:
Does `--help` print the defaults? If so, it would be nice if the help message reflected the value of the environment variable and the Boto config.